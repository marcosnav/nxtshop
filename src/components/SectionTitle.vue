<template>
  <h1 class="SectionTitle">
    <slot />
  </h1>
</template>

<script lang="ts">
import Vue from 'vue';
export default Vue.extend({});
</script>

<style lang="sass" scoped>
.SectionTitle
  font-size: 1.2em
  line-height: 1.728em
  margin-bottom: 0
  position: relative
</style>
