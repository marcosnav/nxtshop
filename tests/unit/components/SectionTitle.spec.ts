import { shallowMount } from '@vue/test-utils';
import SectionTitle from '@/components/SectionTitle.vue';

describe('<SectionTitle>', () => {
  const component = shallowMount(SectionTitle, {
    slots: {
      default: 'SmartWatch X',
    },
  });

  test('renders properly', () => {
    expect(component.classes('SectionTitle')).toBeTruthy();
    expect(component.text()).toMatch('SmartWatch X');
    expect(component.element).toMatchSnapshot();
  });
});
