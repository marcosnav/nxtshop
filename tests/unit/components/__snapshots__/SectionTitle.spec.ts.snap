// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTitle> renders properly 1`] = `
<h1
  class="SectionTitle"
>
  SmartWatch X
</h1>
`;
